chart/inlets-operator/templates/crd.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
The CRD should not be in the chart, it needs to be applied separately and was left out deliberately.

Reply:
Sure I'll change this.